WordPress/src/main/res/values/styles.xml

```diff
@@ -559,4 +559,13 @@
         <item name="android:textSize">@dimen/text_sz_medium</item>
         <item name="android:textStyle">bold</item>
     </style>
+
+    <style name="UploadProgressBar" parent="Widget.AppCompat.ProgressBar.Horizontal">
+        <item name="android:indeterminateOnly">false</item>
+        <item name="android:progressDrawable">@drawable/wp_progress_horizontal</item>
```

Comment:
Good observation! I wanted to make sure we stick to what the design mockups outlined, so I set the colours that way.
As to the `minHeight/maxHeight/scaleY` I needed to set them so there wasn't any natural padding (that would otherwise make it look different than the design), but that said, I have to say this is not working for me on a Samsung Galaxy S3 (I can't see the progressBar), do you have any suggestions to make it work exactly the same? (also not sure whether this is just for that model or it affects all pre-lollipop, for instance)